README.md

```diff
@@ -6,20 +6,21 @@
 
 * Python implementation of the [ENS Name Normalization Standard](https://github.com/adraffy/ensip-norm/blob/main/draft.md).
   Thanks to [Adraffy](https://github.com/adraffy) for his leadership in coordinating the definition of this standard with the ENS community.
-* Passes **100%** of the [official validation tests](https://github.com/adraffy/ens-normalize.js/tree/main/validate) (validated automatically with pytest, see below).
+* Passes **100%** of the [official validation tests](https://github.com/adraffy/ens-normalize.js/tree/main/validate) (validated automatically with pytest on Linux, MacOS, and Windows, see below for details).
 * Passes an [additional suite of further tests](/tools/updater/update-ens.js#L54) for compatibility with the official [Javascript reference implementation](https://github.com/adraffy/ens-normalize.js) and code testing coverage.
 * Based on [JavaScript implementation version 1.9.0](https://github.com/adraffy/ens-normalize.js/tree/4873fbe6393e970e186ab57860cc59cbbb1fa162).
 
 ## Glossary
 
-* name - a full domain name, e.g. `nick.eth`
-* label - a part of a name separated by a dot, e.g. `nick` and `eth` are labels in `nick.eth`
-* normalized name - name that is already in normalized form according to the ENS Normalization Standard
-* normalizable name - name that is normalized or that can be converted into a normalized name using `ens_normalize`
-* disallowed name - name that is not normalized or normalizable
-* curable name - name that may be disallowed but can still be converted into a normalized name using `ens_cure`
-* fatal error - a `DisallowedNameError` object thrown by `ens_normalize` that contains only general information about the error and no possible fixes
-* curable error - a `CurableError` object (inherits from `DisallowedNameError`) thrown by `ens_normalize` that contains information about a possible fix for the error
+* label - a unicode string that does not contain a `.` (full stop / period) character, e.g.  `nick` or `eth`.
+* name - a series of labels separated by a `.` (full stop / period) character, e.g. `nick.eth`.
+* normalized name - name that is in normalized form according to the ENS Normalization Standard. This means the name is equal to itself when using `ens_normalize`.
```

Comment:
> ... name is equal to itself ...

Well, of course it is 😄. I get the point but it sounds slightly confusing (not a native speaker so maybe I'm wrong).
I prefer the more formal `name == ens_normalize(name)`. There should be a way to write it with natural language.